src/components/ConfirmModalForm.js

```diff
@@ -22,12 +23,17 @@ const validate = (values, props) => {
     errors[inputName] = "required";
   }
   else if(inputName.includes("email")) {
-    if (!emailUUIDFormatPattern .test(values.emailConfirmDialogControl)){
+    if (!emailUUIDFormatPattern.test(values.emailConfirmDialogControl)){
       errors[inputName] = "emails.invalid_code";
     }
   }
   else if(inputName.includes("phone")) {
-    if (!phoneLengthPattern.test(values.phoneConfirmDialogControl)){
+    if (!shortConfirmCodeLength.test(values.phoneConfirmDialogControl)){
+      errors[inputName] = "mobile.confirm_code_wrong_length";
+    }
+  }
+  else if(inputName.includes("letter")) {
+    if (!shortConfirmCodeLength.test(values.letterConfirmDialogControl)){
       errors[inputName] = "mobile.confirm_code_wrong_length";
```

Comment:
I really like that the regex variable has been renamed from a specific "phone" name to a generic "confirm code" name!
* Is there a specific reason why the regex has `short` in its name, does it not trigger when codes are longer than 12 chars?
* Could we do something similar to the naming of `mobile.confirm_code_wrong_length` if it applies to both the phone and letter confirmation codes?